src/components/App/App.jsx

```diff
@@ -35,6 +35,7 @@ const App = () => {
                     onOptionClick={step.onOptionClick}
                     nextRoute={steps[stepIndex + 1]?.route ?? "/match"}
                     previousRoute={steps[stepIndex - 1]?.route ?? "/"}
+                    startRoute={steps[0]?.route ?? "/"}
```

Comment:
No need for the fallback value:
```suggestion
                    startRoute={steps[0].route}
```

Reply:
You still need the prop, it's only the `?? "/"` which is unnecessary. `steps[0].route` will always exist.